smk/assembly.smk

```diff
@@ -120,9 +120,19 @@ if config['MEGAHIT_threads'] is None:
 elif type(config['MEGAHIT_threads']) != int:
     print('ERROR in ', config_path, ': MEGAHIT_threads variable is not an integer. Please, complete ', config_path)
 
-if config['MEGAHIT_presets'] is None:
+if config['MEGAHIT_presets'] is None and config['MEGAHIT_custom_k_list'] is None:
     print('ERROR in ', config_path, ': MEGAHIT_presets list of MEGAHIT parameters to run per co-assembly is empty. Please, complete ', config_path)
 
+if 'MEGAHIT_custom_k_list' not in config:
+    config['MEGAHIT_custom_k_list'] = None
+elif config['MEGAHIT_custom_k_list'] is not None:
```

Comment:
Should we check if it is a comma-delimited list of integers, or is it too much and should we just trust the user?

Reply:
I say trust the user, megahit does the checking, so it would just fail to run, if not correct.